test/spec/features/copy-paste/BpmnCopyPasteSpec.js

```diff
@@ -71,6 +71,34 @@ describe('features/copy-paste', function() {
       }));
 
 
+      it('should copy boundary events without host', inject(function(elementRegistry, canvas, copyPaste) {
```

Comment:
Last request: we should have test cases for copying boundary events, pasting them onto root AND paste as attacher.